grails-app/domain/org/pih/warehouse/core/Event.groovy

```diff
@@ -20,12 +21,18 @@ package org.pih.warehouse.core
  *{eventDate: 5/5/2010, eventLocation: Customs, eventType: ARRIVED}*/
 class Event implements Comparable, Serializable {
 
+    def beforeInsert = {
```

Comment:
I remember that we were changing this from closures to methods. I am not sure if the methods work in Grails 1, but if they work, I think it would be better to change them to avoid inconsistency in Grails 3.

Reply:
This is outdated - I had to remove this approach, since triggering an event creates its own session, hence the currentUser was null